inc/os_linux.h

```diff
@@ -9,6 +9,7 @@
 //
 #define _GNU_SOURCE
 #define __USE_GNU
+#define __USE_MISC // needed for SOL_TCP in <netinet/tcp.h>
```

Comment:
I get a warning on strings.h on Ubuntu, strcasecmp not defined, etc.  I will take care of this.